component/button/button.html

```diff
@@ -0,0 +1,144 @@
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="/components/avatar/avatar.css" />
+    <link rel="stylesheet" href="/components/badge/badge.css" />
+    <link rel="stylesheet" href="/components/button/button.css">
+    <title>PineUI | Button</title>
+</head>
+
+<body>
+    <header class="header">
+        <figure class="logo-container">
+          <a href="../index.html">
+            <img src="/assets/logo.png" alt="logo" />
+          </a>
+        </figure>
+        <div class="header-links">
+          <ul class="lg-text">
+            <li><a href="/index.html">Home</a></li>
+           <li><a href="/components/alert/alert.html">Docs</a></li>
+           <li><a href="https://github.com/prajapati-priyanka">Github</a></li>
+          </ul>
+        </div>
+      </header>
+      <main class="component-container wrapper">
+        <aside class="sidenav">
+          <h3 class="sidenav-heading">Components</h3>
+          <ul>
+            <li><a class="md-text" href="../alert/alert.html">Alert</a></li>
+            <li><a class="md-text" href="../avatar/avatar.html">Avatar</a></li>
+            <li><a class="md-text" href="../badge/badge.html">Badge</a></li>
+            <li><a class=" active md-text" href="../button/button.html">Button</a></li>
+            <li><a class="md-text" href="../card/card.html">Card</a></li>
+            <li><a class="md-text" href="../image/image.html">Image</a></li>
+            <li><a class="md-text" href="../input/input.html">Input</a></li>
+            <li><a class="md-text" href="../typography/typography.html">Typography</a></li>
+          </ul>
+        </aside>
+            <section class="comp-display-section">
+                <article class="comp-desc">
+                    <h1 class="comp-heading">Button</h1>
+                    <p class="comp-heading-desc lg-text">
+                        Buttons are also called as call to action. We have range of buttons and their states.
+                    </p>
+                </article>
+
+                <article class="comp-desc">
+                    <h2 class="comp-heading">Solid Buttons</h2>
+                    <p class="comp-heading-desc lg-text">
+                        The solid button component is available in three variants:
+                        <span class="highlight">btn-primary</span>,
+                        <span class="highlight">btn-secondary</span>and
+                        <span class="highlight">btn-danger</span>
+
+                    </p>
+                    <div class="component-code lg-text">
+                        <div class="container-flex">
+
+                            <button class="btn btn-primary">Primary</button>
+                            <button class="btn btn-secondary">Secondary</button>
```

Comment:
You can choose a different color for secondary. It's kind of same as danger color.

Reply:
I will do that. Thanks.